packages/app/src/lib/asset.ts

```diff
@@ -0,0 +1,13 @@
+import { toNumber } from "fuels"
+import { ONE_ASSET_UNIT } from "~/config"
+
+export const calculateRatio = (fromAmount?: bigint | null, toAmount?: bigint | null) => {
+  const _fromAmount = fromAmount || BigInt(0);
```

Comment:
Maybe is safer to use 1 as default instead of 0?

Reply:
I'll test

Reply:
Need to change both